packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -709,3 +714,72 @@ class _CupertinoEdgeShadowPainter extends BoxPainter {
     canvas.drawRect(rect, paint);
   }
 }
+
+/// Displays an iOS-style dialog above the current contents of the app.
+///
+/// This function takes a `builder` which typically builds a [CupertinoDialog]
+/// or [CupertinoAlertDialog] widget. Content below the dialog is dimmed with a
+/// [ModalBarrier]. This widget does not share a context with the location that
```

Comment:
Not clear what you're referring to as `This widget`. 'This' is a static function that returns a Future so it's not clear which 'widget' is being referred to.